cmd/ocm-backplane/cloud/common.go

```diff
@@ -77,6 +78,12 @@ func (cfg *QueryConfig) GetCloudConsole() (*ConsoleResponse, error) {
 		return nil, fmt.Errorf("unable to get token for ocm connection")
 	}
 
+	// Only force traditional flow if explicitly requested via environment variable
+	if os.Getenv("BACKPLANE_FORCE_TRADITIONAL_FLOW") == "true" {
```

Comment:
Do we really need `BACKPLANE_FORCE_TRADITIONAL_FLOW` env vars , what if ovveride AWS proxy based on `BACKPLANE_AWS_PROXY` env var